CelesteNet.Client/Components/CelesteNetChatComponent.cs

```diff
@@ -157,10 +177,28 @@ public CelesteNetChatComponent(CelesteNetClientContext context, Game game)
 
             if (!Active && !isRebinding && Settings.ButtonChat.Button.Pressed) {
                 Active = true;
-
             } else if (Active) {
                 Engine.Commands.Open = false;
 
+                _ControlHeld = MInput.Keyboard.Check(Keys.LeftControl) || MInput.Keyboard.Check(Keys.RightControl);
+
+                if(!MInput.Keyboard.Check(Keys.Left) && !MInput.Keyboard.Check(Keys.Right)) {
```

Comment:
Please change the spacing here to match the rest of the project 😅 

```suggestion
                if (!MInput.Keyboard.Check(Keys.Left) && !MInput.Keyboard.Check(Keys.Right)) {
```